files/lovely-icons/update.json

```diff
@@ -3,6 +3,6 @@
   "name": "lovely-icons",
   "repo": "douglasdeodato/lovely-icons",
   "files": {
-    "include": ["./css/lovely-icons.min.css", "./lovely-icons/*"]
+    "include": ["./css/lovely-icons.min.css", "./lovely-icons/lovely-icons.min.css", "./lovely-icons/*"]
```

Comment:
does `./css/lovely-icons` exist? https://github.com/douglasdeodato/lovely-icons I don't see it